src/NDS.cpp

```diff
@@ -32,7 +32,7 @@
 #include "Wifi.h"
 #include "AREngine.h"
 #include "Platform.h"
-#include "Slot2Cart.h"
+#include "Slot2.hpp"
```

Comment:
I honestly think that the header file should be named "Slot2.h", just to, you know, blend in with the other header files better.